app/models/employer_profile.rb

```diff
@@ -850,6 +850,46 @@ def trigger_notices(event)
     ShopNoticesNotifierJob.perform_later(self.id.to_s, event)
   end
 
+  def upload_document(file_path,file_name,subject,size)
+    #doc_uri = Aws::S3Storage.save(file_path,'id-verification')
+    #file = File.open(file_path, "r:ISO-8859-1")
+
+    tmp_file = "#{Rails.root}/tmp/#{file_name}"
+    id = 0
```

Comment:
Yes, we spoke to Sagar he said, those new bucket names aren't created in order to upload the documents on AWS. So we took this issue to Domnic and he told to write the functionality to store it on local. As per sagar and Domnic suggestions we have implemented the code to upload and store it on local(tmp) in our own way. Once we get those bucket names this method will be removed and we can upload it on cloud @brianweiner.